static/stable/stage/navigation/subscriptions-navigation.json

```diff
@@ -80,6 +80,13 @@
       "title": "Manifests",
       "href": "/subscriptions/manifests"
     },
+    {
+      "id": "cloud-inventory",
+      "appId": "cloud-inventory",
+      "title": "Cloud Inventory",
+      "href": "/subscriptions/cloud-inventory",
```

Comment:
Confirmed this UI is currently deployed to stage and the bundle segment contains a nav-item with this `feoReplacement` ID: https://github.com/RedHatInsights/cloud-inventory-ui/blob/main/deploy/frontend.yaml#L26 :shipit: